.github/workflows/osv-scanner-reusable-pr.yml

```diff
@@ -16,6 +16,14 @@ name: OSV-Scanner PR scanning reusable
 
 on:
   workflow_call:
+    inputs:
+      custom-scan-args:
```

Comment:
nit: can  we just call this "scan-args"? From the user's perspective, "custom" seems unnecessary.

Reply:
Done